### ISSUE TITLE
[ FF7 ] Music and Sound lost passing from field to animation.

### ISSUE THREAD

**Issue by LaZar00** [julianxhokaxhiu/FFNx#83]
Hi. I reported this in Discord:
https://discordapp.com/channels/318179907098116106/391640497954750474/739872387272671283

Hi. At the beginning of FF7, after putting bomb, going up the stairs, going up elevator, opening the two doors, and exiting the reactor, Jessie falls to the ground, and then begins a cinematic with reactor exploding. When the cinematic begins, I lose the sound and music. This happens using MMPEG. When I use original renderer it does not happen. I'm using original .avis extracted from the CDs of the PC98 release. I'm using also MIDI Standard with Yamaha XG and DirectX11. Using 7H 2.2BETA and FFNx Canary 1.7.2.91.

If I save before entering the elevator and follow all the path again it works (with any renderer). Only happens after having putting the bomb and following all the path until exiting the reactor.

Here is the FFNx.log (I have a slow connection):
https://cdn.discordapp.com/attachments/391640497954750474/739887635929235597/FFNx.7z

**Comment by julianxhokaxhiu** [comment]
Hi @LaZar00 can you please try again if this issue is still a thing? We've been working hard on rebuilding from the ground up the Audio Engine and I was curious to know if you can still trigger this issue.

Please make sure you download and use the latest canary release. Be sure to use also the given `FFNx.cfg` file as new switches were added.

Thank you in advance

**Comment by LaZar00** [comment]
Hi julian,

I tested this after install 7H 2.2 Final Release (2.2.3.522), with its own FFNx and it worked well, but with the latest Canary release of FFNx, I don't know if it works. Right now I can not install FFNx canaries, I need to test the game with 7H Final Release to see that it works well as is. But it is very easy to test, it is at the beginning of the game...

**Comment by LaZar00** [comment]
Hi again Julian,

I decided to test the latest 1.7.2.179 Canary release of FFNx (I've made a backup of the one that comes with 7H 2.2).
I passed again from the saving point the mako reactor explosion ending at the tunnel (after animation).

I have NOT any problems with the sound/music. So I would say that this issue is solved. At least the issue I detected when making some preliminary tests of my project.

Thanks a lot.
